ctwrap/fileio/yaml.py

```diff
@@ -1,33 +1,33 @@
-#!/usr/bin/env python
-# -*- coding: utf-8 -*-
 """Function handling file input/output."""
 
 import os
-import re
+from typing import Optional, Dict,Any
```

Comment:
Now that all other modules are deleted, it would make sense to move this file into the main folder? (I.e. one level up)